lib/config_upgrader.js

```diff
@@ -235,8 +236,9 @@ class ConfigUpgrader {
       const config = new Config({
         configFile: configFile,
         cwd: process.cwd()
-      });
-      upgrader.upgrade(config.useSpecificConfig);
```

Comment:
will you please summarize why is it we don't seem to use the `ConfigUpgrader` outside of tests anymore? If we don't need it, we should delete it totally.